src/index.html

```diff
@@ -1,14 +1,36 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>
+      Enclosures
+    </title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="level_one" id="qa-block">
+      <div class="level_two">
+        <div class="level_three">
```

Comment:
have your read checklist? especially the part about class names

Reply:
https://prnt.sc/t9lzks
yes, i had read

(and there are not comments in my html/css)

Reply:
https://github.com/mate-academy/layout_colored-blocks/blob/master/checklist.md
4th from the bottom

Reply:
it's hard to understand what kind of level 1, 2, 3 you meant, name them like outer-box, mid-box and so on. This time I'll know that you have such a structure with several nested blocks